docker-images/build-images.sh

```diff
@@ -7,6 +7,12 @@ MODE=${1:-"push"}
 DOCKER_TAG=${DOCKER_TAG:-"latest"}
 MVN_ARGS=${MVN_ARGS:-""}
 
+echo "Installing root pom"
+make mvn_build
+
+echo "Building Tracing dependency"
+make build --directory=tracing/
+
```

Comment:
I'm not sure I understand why is this needed. Can you elaborate on this a bit?

Reply:
All other modules depend on this one -- so it needs to be built before.

Reply:
Normally, the submodule would be added into the `SUBDIRS` list in the main Makefile. So when you run `make ...` it would trigger it for the tracing module as well and build it that way.

The README.md suggests to call `make all` to build the project. But if this is only in the `build-images.sh` script, it would be never called and I assume it would not work with it. That is at least my read on the build.

Reply:
Atm the script iterates over `kafka` and `http` subdirs, and builds them.
Whereas `tracing` module is a seperate module on which they depend on,
so it must be built before.

This build-images.sh is only used in CI, afais.
The plain Make makes use of `SUBDIRS`, where `tracing` is listed as well.

If @im-konge or @Frawless have any better ideas, fine by me.